src/things_mcp/applescript_bridge.py

```
@@ -95,9 +110,41 @@ def add_todo_direct(title: str, notes: Optional[str] = None, when: Optional[str]
         script_parts.append('  end try')
         script_parts.append('end try')
     
+    # Add to project/area by UUID (takes precedence over list_title)
+    if list_id:
+        escaped_id = escape_applescript_string(list_id)
+        script_parts.append(f'try')
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)